accelerator/apis.h

```diff
@@ -111,7 +112,7 @@ status_t api_receive_mam_message(const iota_client_service_t* const service,
  */
 status_t api_mam_send_message(const iota_config_t* const tangle,
                               const iota_client_service_t* const service,
-                              char const* const payload,
+                              const send_mam_req_t* req,
```

Comment:
Did you update the corresponding descriptions about parameters?

Reply:
I removed the change here. @wusyong  said it's what we going to do next step.

Reply:
I removed the change. we will add the change to `apis.c` after serialiaztion is complete.